hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImpl.java

```diff
@@ -217,21 +218,27 @@ public PreHandleResult preHandleTransaction(
             // In that case, the payer will end up paying for the transaction. So we still need to do the signature
             // verifications that we have determined so far.
             final var results = signatureVerifier.verify(txInfo.signedBytes(), expanded);
-            return preHandleFailure(payer, payerKey, preCheck.responseCode(), txInfo, results);
+            return preHandleFailure(payer, payerKey, preCheck.responseCode(), txInfo, Set.of(), results);
```

Comment:
Perhaps use `Collections.emptySet()` here instead of `Set.of()`

Reply:
What would be the advantage? I am using `Set.of(...)` to define all kinds of immutable sets. It seems natural to use it also for empty sets.